### ISSUE TITLE
Three Blocks Tall?

### ISSUE THREAD

**Issue by Drakynnan** [kingrunes/Somnia#26]
If playing on a LAN server, the mod works fine except for when a player wakes. It seems that whatever player was sleeping then cannot walk through areas that are only two blocks tall anymore. It seems that the mod causes the player to seemingly become three blocks tall upon waking. As I said, I've only encountered this bug when playing on LAN, and only when another player has joined.

**Comment by callumerrington** [comment]
This is a strange bug which keeps coming back now and then. I can repeat it myself, but still I have no idea why two people joining a LAN server would effect the player's height. Have you got any other mods installed when this bug occurs?

**Comment by fishb0nes** [comment]
There are a lot of other mods installed, but the player doesn't actually become three blocks tall, it's just that they can no longer pass under anything of two blocks high. It has only occured on LAN servers with more than one player

**Comment by callumerrington** [comment]
I tried launching two instances of the game, loading my single player world then opening to LAN and joining with the second instance. I placed two beds, woke both players one at a time, I built a door frame-like structure only two blocks high and both the host player and second player were able to walk through.

What steps did you take to produce this?

**Comment by fishb0nes** [comment]
I don't think that I've tried waking each player one by one, but I remember this error occurring when the player wakes "naturally." I've tried having one player sleep, and the other join whilst one is sleeping, but the error still persists for only the one that was sleeping

**Comment by callumerrington** [comment]
Can you do that on the latest version for 1.7.10? I haven't been able to reproduce this.

**Comment by fishb0nes** [comment]
That's where I was experiencing it
